trunk/SUAVE/Attributes/Solids/Acrylic.py

```diff
@@ -0,0 +1,29 @@
+# Acrylic.py
+#
+# Created: Jul 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from Solid import Solid
+from SUAVE.Core import Data
+
+#-------------------------------------------------------------------------------
+# Acrylic Solid Class
+#-------------------------------------------------------------------------------
+
+class Acrylic(Solid):
+
+    """ Physical Constants Specific to Acrylic"""
+
+    def __defaults__(self):
+
+        self.UTS        = 75e6          # Ultimate Tensile Strength
+        self.USS        = 55.2e6        # Ultimate Shear Strength
+        self.UBS        = 0.0           # Ultimate Bearing Strength
```

Comment:
Please use units, for example * Units.Pascals.
Echo what Tim said, you need descriptive names. Leaving a comment next to it isn't enough, when i'm viewing the results in another script I won't know what UBS is.

Reply:
Changed to verbose names in compliance with naming convention and added units